### ISSUE TITLE
TypeError: Cannot read property 'unshift' of undefined at PluginPass.module.exports.Transformer.Program

### ISSUE THREAD

**Issue by garytryan** [speedskater/babel-plugin-rewire#118]
Full Error: 

```
Module build failed: TypeError: /Users/garyryan/Documents/quill4/tests/state/story-list.test.js: Cannot    read property 'unshift' of undefined at PluginPass.module.exports.Transformer.Program (/Users/garyryan/Documents/quill4/node_modules/babel-rewire-plugin/src/babel-rewire-plugin.js:55:12)
```

Webpack setup:

```
module: {
		loaders: [
			{
				test: /\.jsx?$/,
				exclude: /(node_modules)/,
				loader: 'babel',
				query: {
					presets: [ 'es2015', "stage-0", 'react' ],
					plugins: ['babel-rewire-plugin']
				}
			}
		]
}
```
I'm using Babel 6.5.1, in case that's relevant.

Any ideas what's going wrong here?

**Comment by speedskater** [comment]
@garytryan Thanks for reporting the issue. Could you please create a small sample project reproducing the error.

**Comment by speedskater** [comment]
@garytryan I think your issue should be fixed in 1.0.0. Hence, I will close this issue. If the error still occurs please reopen this issue with a detailed description.
